### PR TITLE
Fix Cannot read property 'id' of undefined

### PR DESCRIPTION
**Pull request by kvas-damian**
Fixes: https://sentry.io/organizations/wnl-production/issues/885120452/?project=1207095&query=is%3Aunresolved&statsPeriod=14d

### CODE REVIEW

File: resources/assets/js/components/quiz/QuizQuestion.vue [hunk 1]
```diff
@@ -109,8 +109,12 @@
 						@switchSlide="changeSlide"
 						@userEvent="onRelatedSlideUserEvent"
 					>
-						<span slot="header">{{slideLink(currentModalSlide)}}</span>
+						<span
+							v-if="currentModalSlide.context.id"
```
Comment (rogatty) [inline resources/assets/js/components/quiz/QuizQuestion.vue]: When is it empty?

Comment (kvas-damian) [reply resources/assets/js/components/quiz/QuizQuestion.vue]: It's empty when it's not assigned to a presentation.

Comment (rogatty) [reply resources/assets/js/components/quiz/QuizQuestion.vue]: Can't we move it to this line, then?
```
<wnl-slide-preview
	v-if="slideContent && currentModalSlide.id"
```

Comment (kvas-damian) [reply resources/assets/js/components/quiz/QuizQuestion.vue]: Then the UX would be very strange. You click on a slide preview link and nothing happen - the modal won't be shown.

**Pull request merged**